norse/benchmark/norse_lif.py

```diff
@@ -22,46 +22,47 @@ def forward(
         self, input_spikes: torch.Tensor, p: LIFParametersJIT, s: LIFFeedForwardState
     ):
         sequence_length, batch_size, features = input_spikes.shape
-
-        inputs = input_spikes.unbind(0)
         # spikes = torch.jit.annotate(List[Tensor], [])
-        spikes = torch.empty((sequence_length, batch_size, features))
+        spikes = torch.empty(
+            (sequence_length, batch_size, features), device=input_spikes.device
+        )
 
-        for ts in range(len(inputs)):
-            x = self.fc(inputs[ts])
+        for ts in range(sequence_length):
+            x = self.fc(input_spikes[ts])
             z, s = _lif_feed_forward_step_jit(input_tensor=x, state=s, p=p, dt=self.dt)
             spikes[ts] = z
 
         return spikes
 
 
 def lif_feed_forward_benchmark(parameters: BenchmarkParameters):
-    model = LIFBenchmark(parameters).to(parameters.device)
-    input_spikes = PoissonEncoder(parameters.sequence_length, dt=parameters.dt)(
-        0.3
-        * torch.ones(
-            parameters.batch_size, parameters.features, device=parameters.device
+    with torch.no_grad():
```

Comment:
Have you checked whether this actually improves runtime performance?

Reply:
Yeah, in my latest benchmarks it actually increases the performance by ~20% to leave the gradients out. Oddly enough, because I do remember we had conflicting earlier results.